ui/rewards/steps/CongratsStepContent.tsx

```diff
@@ -0,0 +1,124 @@
+import { Text, Box, Flex, Button, Skeleton, Icon } from '@chakra-ui/react';
+import React from 'react';
+
+import { route } from 'nextjs-routes';
+
+// This icon doesn't work properly when it is in the sprite
+// Probably because of the gradient
+// eslint-disable-next-line no-restricted-imports
+import meritsIcon from 'icons/merits_colored.svg';
+import { useRewardsContext } from 'lib/contexts/rewards';
+import IconSvg from 'ui/shared/IconSvg';
+
+import CopyField from '../CopyField';
+
+type Props = {
+  isReferral: boolean;
+}
+
+const CongratsStepContent = ({ isReferral }: Props) => {
+  const { referralsQuery, rewardsConfigQuery } = useRewardsContext();
+
+  const registrationReward = rewardsConfigQuery.data?.rewards.registration;
+  const referralReward = Number(rewardsConfigQuery.data?.rewards.registration_with_referral) - Number(rewardsConfigQuery.data?.rewards.registration);
+  const refLink = `https://eth.blockscout.com?ref=${ referralsQuery.data?.code }`;
+  const shareText = `Just signed up for @blockscoutcom Merits program and got ${ registrationReward } merits!\n\nUse my referral link to get extra ${ referralReward } merits: ${ refLink }`; // eslint-disable-line max-len
+  return (
+    <>
+      <Flex
+        alignItems="center"
+        background="linear-gradient(254.96deg, #9CD8FF 9.09%, #D0EFFF 88.45%)"
+        borderRadius="md"
+        padding={ 2 }
+        mb={ 8 }
+      >
+        <Icon as={ meritsIcon } boxSize="60px" mb={ -1 }/>
+        <Skeleton isLoaded={ !rewardsConfigQuery.isLoading }>
+          <Text fontSize="30px" fontWeight="700" color="blue.700">
+            +{ rewardsConfigQuery.data?.rewards[ isReferral ? 'registration_with_referral' : 'registration' ] }
+          </Text>
+        </Skeleton>
+        { isReferral && (
+          <Flex alignItems="center" h="56px">
+            <Box w="1px" h="full" bgColor="whiteAlpha.800" mx={ 8 }/>
+            <Flex flexDirection="column" justifyContent="space-between">
+              { [
+                {
+                  title: 'Registration',
+                  value: registrationReward,
+                },
+                {
+                  title: 'Referral program',
+                  value: referralReward,
+                },
+              ].map(({ title, value }) => (
+                <Flex key={ title } alignItems="center">
+                  <Icon as={ meritsIcon } boxSize={ 8 } mb={ -0.5 }/>
+                  <Skeleton isLoaded={ !rewardsConfigQuery.isLoading }>
+                    <Text fontSize="sm" fontWeight="700" color="blue.700">
```

Comment:
Should this color be the same in the dark mode?

Reply:
Same question about all other colors.

Reply:
There was no design for a dark mode for this component 🥲

Reply:
Please ask the designer to add one 😇